pkg/apis/core/types_shoot.go

```diff
@@ -335,6 +335,8 @@ type ClusterAutoscaler struct {
 	MaxNodeProvisionTime *metav1.Duration
 	// MaxGracefulTerminationSeconds is the number of seconds CA waits for pod termination when trying to scale down a node (default: 600).
 	MaxGracefulTerminationSeconds *int32
+	// IgnoreTaint specifies a taint to ignore in node templates when considering to scale a node group
```

Comment:
a list of taints?

Reply:
Actually, a list of taint keys, or?

Reply:
Yeah, 'list of taint keys' would also make more sense. Thanks!